sqlalchemy-stubs/orm/decl_api.pyi

```diff
@@ -46,42 +52,51 @@ class _stateful_declared_attr(declared_attr):
     def __call__(self, fn: Any): ...
 
 def declarative_base(
-    bind: Optional[Any] = ...,
-    metadata: Optional[Any] = ...,
-    mapper: Optional[Any] = ...,
-    cls: Any = ...,
+    bind: Optional[
+        Any
+    ] = ...,  # NOTE: Deprecated in 1.4, to be removed in 2.0.
+    metadata: Optional[MetaData] = ...,
+    mapper: Optional[Mapper] = ...,
+    cls: Union[type, Tuple[type, ...]] = ...,
     name: str = ...,
-    constructor: Any = ...,
-    class_registry: Optional[Any] = ...,
-    metaclass: Any = ...,
-): ...
+    constructor: Callable[..., None] = ...,
+    class_registry: Optional[MutableMapping[Any, Any]] = ...,
+    metaclass: type = ...,
+) -> type: ...
 
 class registry:
-    metadata: Any = ...
-    constructor: Any = ...
+    metadata: MetaData
+    constructor: Callable[..., None]
     def __init__(
         self,
-        metadata: Optional[Any] = ...,
-        class_registry: Optional[Any] = ...,
-        constructor: Any = ...,
+        metadata: Optional[MetaData] = ...,
+        class_registry: Optional[MutableMapping[Any, Any]] = ...,
+        constructor: Callable[..., None] = ...,
         _bind: Optional[Any] = ...,
     ) -> None: ...
     @property
-    def mappers(self): ...
+    def mappers(self) -> FrozenSet[Mapper]: ...
     def configure(self, cascade: bool = ...) -> None: ...
     def dispose(self, cascade: bool = ...) -> None: ...
     def generate_base(
         self,
-        mapper: Optional[Any] = ...,
-        cls: Any = ...,
+        mapper: Optional[Mapper] = ...,
```

Comment:
this is a Callable that would return Mapper objects.   i dont even know why we have this but it's still there.